tests/conftest.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+"""
+You can create a file called `.env` in the root of the repo, containing your local env vars.
+"""
+from dotenv import load_dotenv
+
+# Load environment variables
+# needs to happen before anything else (to properly instantiate constants)
+
+load_dotenv(verbose=True)
+
+import os
+import pytest
+import json
+import requests
+import uuid
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+from mailchimp3.mailchimpclient import MailChimpError
+
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk import Tracker
+from rasa.shared.core.domain import Domain
```

Comment:
I think we want to use the rasa_sdk `Domain` here, which is already a dict

Reply:
It is, but we want to load the actual domain of the bot - I started using an empty dict, but realized that could mask problems in the case that an action starts looking in the domain.